gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-all.zip
```

Comment:
I am almost 100% sure that this change should be reverted. Mobile team is using a build server and they don’t change the version of this as often as developers do in our local machines.

From what I have seen, there is no need to update Gradle or Android Gradle Plugin.

Reply:
Will do!